File: app/src/main/java/net/emojiparty/android/jishotomo/ui/presentation/ScreenSize.java
package net.emojiparty.android.jishotomo.ui.presentation;

import android.content.res.Resources;
import android.util.TypedValue;

// http://android.pcsalt.com/set-margins-in-dp-programmatically-android/
public class ScreenSize {
  public static int dpsToPixels(float dps, Resources resources) {
    return (int) TypedValue.applyDimension(TypedValue.COMPLEX_UNIT_DIP, dps,
        resources.getDisplayMetrics());
  }
}

File: app/src/main/java/net/emojiparty/android/jishotomo/ui/presentation/ScreenSize.kt
package net.emojiparty.android.jishotomo.ui.presentation

import android.content.res.Resources
import android.util.TypedValue

// http://android.pcsalt.com/set-margins-in-dp-programmatically-android/
object ScreenSize {
  fun dpsToPixels(
    dps: Float,
    resources: Resources
  ): Int {
    return TypedValue.applyDimension(
        TypedValue.COMPLEX_UNIT_DIP, dps,
        resources.displayMetrics
    ).toInt()
  }
}